3rdparty/libturbojpeg/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+#  CMake file for libjpeg. See root CMakeLists.txt
+#
+# ----------------------------------------------------------------------------
+project(${JPEG_LIBRARY})
+
+ocv_include_directories(${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
+
+set(LIB_TURBOJPEG_SOURCES 
+  jaricom.c jcarith.c jdarith.c
+  jcapimin.c jcapistd.c jccoefct.c jccolor.c 
+	jcdctmgr.c jchuff.c jcinit.c jcmainct.c jcmarker.c jcmaster.c 
```

Comment:
Consistent indentation. Also for the rest of file - some of the indentations are done using tabs and some using spaces.

Reply:
Done.